src/main/java/org/cactoos/bytes/HexOf.java

```diff
@@ -72,7 +73,7 @@ public HexOf(final Text origin) {
                 }
                 return result;
             },
-            hex.chars().mapToObj(c -> (char) c).iterator()
+            new IterableOf<>(hex.chars().mapToObj(c -> (char) c).iterator())
```

Comment:
@mazdack it would be cleaner that IterableOf takes a Stream instead (which would then call `iterator` of course)